cluster/watchdog.go

```diff
@@ -121,6 +121,9 @@ func (w *Watchdog) rescheduleContainers(e *Engine) {
 			}
 		}
 
+		// Clear out the network configs, since we cannot create a
+		// container with multiple networks.
+		c.Config.NetworkingConfig.EndpointsConfig = nil
```

Comment:
That's correct, but it'll get updated when we do the `NetworkConnect` calls. The reason why we can't create the new container with the `EndpointsConfig` object is that you can't create a container that's attached to multiple networks.